src/components/head-metadata/index.tsx

```diff
@@ -46,13 +52,13 @@ export default function HeadMetadata(props: HeadMetadataProps) {
 	return (
 		// TODO: OpenGraph image to be passed as the image prop here
 		<>
-			<SelfReferentialCanonicalTag />
```

Comment:
Some strange interaction with the usage of a nested `<Head>` tag here, pulling it out and passing the URL directly to `HashiHead` fixes the duplicate tag 🤷‍♂️